### PR TITLE
Hoes

### PR DESCRIPTION
**Pull request by SaiMonto**
Added the hoes

### CODE REVIEW

**Comment by OmniacDev** [comment]
Bump the manifest versions up to 1.0.10

**Comment by SaiMonto** [comment]
version added

**Review by OmniacDev — CHANGES_REQUESTED** [review]

File: packs/RP/models/entity/attachable/geometry.enderverse_hoe.json
Comment (OmniacDev) [inline packs/RP/models/entity/attachable/geometry.enderverse_hoe.json:1]: `ev_hoe.geometry.json`

File: packs/RP/animations/attachables/ameythst.animation.json
Comment (OmniacDev) [inline packs/RP/animations/attachables/ameythst.animation.json:1]: `ev_hoe.animation.json`

**Review by OmniacDev — CHANGES_REQUESTED** [review]
Rename these files

**Review by OmniacDev — APPROVED** [review]
LGTM

**Pull request merged**